packages/devextreme-scss/scss/widgets/generic/chat/_sizes.scss

```diff
@@ -51,12 +64,17 @@ $chat-messagelist-day-header-first-padding-top: 20px !default;
   $chat-errors-row-gap: 4px !default;
   $chat-error-padding-block: 9px !default;
 
-  $chat-messagelist-empty-icon-box-size: 64px !default;;
+  $chat-messagelist-empty-icon-box-size: 64px !default;
```

Comment:
Please fix this typo in others file also 
<img width="507" alt="image" src="https://github.com/user-attachments/assets/20487590-6dae-4a68-b59c-12ab313ae3c7">
Thank you for your attention

Reply:
[536fec0c8bc89a7afd1a00a7114ead566de697b6]